src/new-client/src/Web/TearOutSection/TearOutComponent.tsx

```diff
@@ -0,0 +1,35 @@
+import { HeaderAction } from "@/components/styled"
+import { PopOutIcon } from "@/components/icons/PopOutIcon"
+import { tearOutSection } from "@/Web/TearOutSection/state"
+import { PopInIcon } from "@/components/icons/PopInIcon"
+import {
+  TornOutTileContext,
+  TornOutTradeContext,
+  TornOutAnalyticsContext,
+} from "@/Web/context"
+
+export const TearOutComponent: React.FC<{ section: string }> = ({
+  section,
+}) => {
+  var context = TornOutTileContext
+  if (section === "trades") {
+    context = TornOutTradeContext
+  } else {
+    context = TornOutAnalyticsContext
+  }
+  return (
+    <context.Consumer>
```

Comment:
You can use the `useContext` hook, it's much nicer to work with than this api.
https://reactjs.org/docs/hooks-reference.html#usecontext

Reply:
Done